site/docs/v1/tech/email-templates/templates-replacement-variables.adoc

```diff
@@ -520,6 +522,110 @@ The Tenant object, see the link:/docs/v1/tech/apis/tenants[Tenant API] for field
 [field]#user# [type]#[User]#::
 The User object, see the link:/docs/v1/tech/apis/users[User API] for field definitions of a User.
 
+== Two Factor Authentication Method Added
+
+include::docs/v1/tech/shared/_enterprise-edition-blurb.adoc[]
+
+.HTML
+[source,text]
+----
+<p>
+The following two factor method was added to ${user.email}:
+
+    <br>
+    <strong>Method: ${method.method}</strong>
+    <br>
+    <strong>Identifier: ${method.id}</strong>
+
+</p>
+
+- FusionAuth Admin
+----
+
+.Text
+[source,text]
+----
+The following two factor method was added to ${user.email}:
+
+- Method: ${method.method}
+- Identifier: ${method.id}
+
+- FusionAuth Admin
+----
+
+=== Replacement Variables
+
+[.api]
+[field]#application# [type]#[Application]# [since]#Available since 1.28.0#::
+The Application object, see the link:/docs/v1/tech/apis/applications[Application API] for field definitions.
++
+*Note*:
+This object may not be available depending upon when this template is constructed. If you utilize this object in your template, ensure you first check to see if it is defined.
```

Comment:
Can we put an example of how to check for that? I assume it is some freemarker: `${(application)!""}` for example.